azure-storage-cpp-lite/src/storage_credential.cpp

```diff
@@ -76,9 +86,46 @@ namespace microsoft_azure {
             return url;
         }
 
-        void shared_access_signature_credential::sign_request(const storage_request_base &, http_base &h, const storage_url &, const storage_headers &) const {
+        void shared_access_signature_credential::sign_request(const storage_request_base &,
+                http_base &h,
+                const storage_url &,
+                const storage_headers &) const
+        {
             std::string transformed_url = transform_url(h.get_url());
             h.set_url(transformed_url);
         }
+
+        AZURE_STORAGE_API token_credential::token_credential(){
+            m_credmgr_ptr = GetTokenManagerInstance(EmptyCallback);
+        }
```

Comment:
TODO: Add comment expecting a token manager instance to already have been initialized, for usability purposes.